application/dialogs/NewInstanceDialog.cpp

```diff
@@ -122,13 +124,15 @@ QList<BasePage *> NewInstanceDialog::getPages()
 {
     importPage = new ImportPage(this);
     twitchPage = new TwitchPage(this);
+    auto technicPage = new TechnicPage(this);
     return
     {
         new VanillaPage(this),
         importPage,
         new FtbPage(this),
         new LegacyFTB::Page(this),
-        twitchPage
+        twitchPage,
```

Comment:
I've been keeping to alphabetical order for the modpack sources, ideally Technic would come before Twitch.